web-app/CKS.Web/appsettings.json

```diff
@@ -18,6 +18,6 @@
     "UseRabbit": false,
     "UseFile": true
   },
-  "SearchApiUrl": "https://test-search-api.nice.org.uk",
+  "SearchApiUrl": "http://test-search-api.nice.org.uk",
```

Comment:
This shouldn't affect going to live as http will redirect to https.

Reply:
yep agree